src/screens/ReaderScreen.tsx

```diff
@@ -1,13 +1,27 @@
 import * as React from "react";
 import { FunctionComponent } from 'react';
-import { SafeAreaView, StyleSheet } from 'react-native';
+import { Dimensions, SafeAreaView, StyleSheet } from 'react-native';
 import { ReaderScreenProps } from '../navigationTypes';
 import BookReader from "../BookReader";
 
+const windowDimensions = Dimensions.get('window');
+
 export const ReaderScreen: FunctionComponent<ReaderScreenProps> = ({ route, navigation }: ReaderScreenProps) => {
+    const [windowSize, setWindowSize] = React.useState(windowDimensions);
+
+    React.useEffect(() => {
+        const subscription = Dimensions.addEventListener(
```

Comment:
https://reactnative.dev/docs/usewindowdimensions is the preferred way now

Reply:
Thanks for finding that! Much cleaner!